components/OrgCard.tsx

```diff
@@ -0,0 +1,22 @@
+import Link from 'next/link';
+import { Org } from 'interfaces';
```

Comment:
need to change all instances of `Org` interfaces to `Organization`

Reply:
Ended up changing all instances to OrgApp.